cmd/lncli/commands.go

```diff
@@ -628,6 +625,20 @@ func openChannel(ctx *cli.Context) error {
 	}
 }
 
+func decodeNodePubkeyOrURI(val string) ([]byte, string, error) {
+	var nodeKey, connectVal string
+	split := strings.Split(val, "@")
+
+	if len(split) > 1 {
+		nodeKey, connectVal = split[0], split[1]
```

Comment:
looks like `nodeKey` is never set in the case where no URI is given?

Reply:
D'oh! Good find. Willfix.